platform/darwin/src/MGLLight.h

```diff
@@ -117,15 +127,19 @@ MGL_EXPORT
 /**
  Color tint for lighting extruded geometries.
  
- The default value of this property is an `MGLStyleValue` object containing
+ The default value of this property is an expression that evaluates to
  `UIColor.whiteColor`.
  
- You can set this property to an instance of:
+ You can set this property to an expression containing any of the following:
+ 
+ * Constant values
+ * Predefined functions, including mathematical and string operators
+ * Conditional expressions
+ * Variable assignments and references to assigned variables
+ * Interpolation and step functions applied to the `$zoomLevel` variable
  
- * `MGLConstantStyleValue`
- * `MGLCameraStyleFunction` with an interpolation mode of:
-   * `MGLInterpolationModeExponential`
-   * `MGLInterpolationModeInterval`
+ This property does not support applying interpolation or step functions to
+ feature attributes.
```

Comment:
Note that, other than documentation, this PR hasn’t migrated light properties to expressions yet.